2_setup_my_env_as_user.ubuntu.sh

```
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+echo "🐍 Setting up Python environment..."
+
+python3 -m venv .venv
+source .venv/bin/activate
```

Shellcheck (0.10.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)